content/docs/pipelines/installation/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Installing Pipelines"
```

Comment:
It's great to have such page to explain all options. Thanks!

Reply:
Thanks @jessiezcc!